### PR TITLE
PE-4155: design inconsistencies turbo top up

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/4c0j5bftsj4to
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/4ler4fr4mmu90

### CODE REVIEW

File: lib/components/top_up_dialog.dart [hunk 1]
```diff
@@ -424,46 +424,71 @@ class _PresetAmountSelectorState extends State<PresetAmountSelector> {
...
+            mobile: (_) => Stack(
               children: [
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    const SizedBox(height: 24),
+                    Text(
+                      _customAmountValidationMessage == null &&
+                              (_customAmountValidationMessage?.isEmpty ?? true)
+                          ? 'Custom Amount (min \$10 - max \$10,000)'
```
Comment (matibat) [inline lib/components/top_up_dialog.dart:462]: Should it be localized?

File: lib/l10n/app_en.arb
Comment (matibat) [inline lib/l10n/app_en.arb:1]: Are these changes coming from a Localizely export?

**Review by matibat — COMMENTED** [review]
LGTM - I'll give it a quick test, and then approve

File: lib/l10n/app_en.arb
Comment (thiagocarvalhodev) [reply lib/l10n/app_en.arb:1]: Yes

**Review by matibat — APPROVED** [review]
I mistakenly applied a commit here, so I reverted. Re-approving because there are no changes.

**Comment by matibat** [comment]
The commit I was talking about regards to this PR #1208 (PE-4187)

**Pull request merged**